statistics/update_test.go

```diff
@@ -456,53 +456,45 @@ func (s *testStatsUpdateSuite) TestQueryFeedback(c *C) {
 			// test primary key feedback
 			sql: "select * from t where t.a <= 5",
 			hist: "column:1 ndv:3 totColSize:0\n" +
-				"num: 1\tlower_bound: 1\tupper_bound: 1\trepeats: 1\n" +
+				"num: 1\tlower_bound: -9223372036854775808\tupper_bound: 1\trepeats: 0\n" +
```

Comment:
This negative value is expected?

Reply:
Yes, for `t.a <= 5`, the scan range is `[MinInt64, 5]`.